output/subclass-super-plug-category-hashes.json

```diff
@@ -0,0 +1,17 @@
+[
```

Comment:
README.md documents these output files in a table, wouldn't hurt to add this one too.